src/System.Private.ServiceModel/tests/Scenarios/Encoding/Encoders/TextEncodingTests.cs

```diff
@@ -48,4 +48,36 @@ public static void SameBinding_SecurityModeNone_Text_EchoString_Roundtrip()
 
         Assert.True(errorBuilder.Length == 0, "Test case FAILED with errors: " + errorBuilder.ToString());
     }
+
+    [Fact]
+    [OuterLoop]
+    public static void TextMessageEncoder_WrongContentTypeResponse_Throws_ProtocolException()
+    {
```

Comment:
All our other tests are known good content types. Returning the wrong content type proves we are examining the content type and throw an exception if it's wrong. Not throwing an exception on all other tests shows we accept a good value.